src/custom_skills/VectorEmbed/__init__.py

```diff
@@ -26,12 +27,19 @@ def function_vector_embed(req: func.HttpRequest) -> func.HttpResponse:
     except jsonschema.exceptions.ValidationError as e:
         return func.HttpResponse("Invalid request: {0}".format(e), status_code=400)
 
+    managed_identity_client_id = os.environ.get("MANAGED_IDENTITY_CLIENT_ID")
+
+    credential = DefaultAzureCredential(managed_identity_client_id=managed_identity_client_id)
```

Comment:
so you were saying something about Manage Identites not woking For AI Search, might leave a comment here or a TODO: to exxplain what is going on, I know you have an upcoming PR coming up, but just a TODO: so that we do not forget

Reply:
User Managed Identity works fine if you are using it to interact with AI Search. It doesn't work if it's a part of your index (vectorizer), and the index should access external resources using it. So, we are using System Assigned Identity. It's going to be a big document about it, and I will add a reference to it here once it's available.